### PR TITLE
Replace Makefile to nodejs postinstall script

### PR DESCRIPTION
**Pull request by tormozz48**
@tadatuta @tavriaforever

### CODE REVIEW

**Comment by tadatuta** [comment]
@tormozz48 looks quite ugly.
can't we just use `npm` commands without `post-install.js`?
like:

``` js
"scripts": {
    "start": "node server/index.js",
    "deps": "bower install",
    "build": "npm run deps && enb make --no-cache",
    "postinstall": "npm run build"
   }
```

**Comment by tormozz48** [comment]
You hurt me :smile: I made this solution with detail logging which is suitable for every woman from countryside. Ok I will change it this way

**Comment by tormozz48** [comment]
@tadatuta Please review again

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "enb-roole": "0.2.1"
   },
   "scripts": {
-    "start": "node server/index.js"
+    "start": "node server/index.js",
+    "postinstall": "bower install && enb make --no-cache && cd configs && ln -snf development current"
```
Comment (tadatuta) [inline package.json]: you call `make` before creating symlink. is it ok?

**Pull request merged**